src/web/lib/hakuneko/frontend@classic-light/jobs.html

```diff
@@ -178,6 +221,41 @@
                 }
             }
 
+            /**
+             *
+             */
+            getPlayPauseTooltip( status ) {
+                if ( status ) {
+                    return 'Resume download queue';
+                } else {
+                    return 'Pause download queue';
+                }
+            }
+
+            /**
+             *
+             */
+            getPlayPauseClass( status ) {
+                if ( status ) {
+                    return 'fa-play';
+                } else {
+                    return 'fa-pause';
+                }
+            }
+
+            /**
+             *
+             */
+            playPauseQueue() {
+                if(this.queuePaused) {
+                    Engine.DownloadManager.startQueue();
+                    this.queuePaused = false;
```

Comment:
In case of `startQueue` failed, the assignment of `false` to `this.queuePaused` is incorrect, use `DownloadManager.isQueuePaused` instead.

```javascript
if(this.queuePaused) {
    Engine.DownloadManager.startQueue();
} else {
    Engine.DownloadManager.pauseQueue();
}
this.queuePaused = Engine.DownloadManager.isQueuePaused;
```